docs/xml/enumerations/vote_variation.rst

```diff
@@ -1,35 +1,43 @@
 VoteVariation
 =============
 
-.. todo::
-
-   Document VoteVariation
 
 +----------------------+----------------------------------------------------------------------------------+
 | Name                 | Description                                                                      |
 |                      |                                                                                  |
 +======================+==================================================================================+
-| 1-of-m               |                                                                                  |
-+----------------------+----------------------------------------------------------------------------------+
-| approval             |                                                                                  |
+| 1-of-m               | A method where each voter can select up to one candidate.                        |
 +----------------------+----------------------------------------------------------------------------------+
-| cumulative           |                                                                                  |
+| approval             | `Approval voting`_, where each voter can select as many candidates as desired.   |
 +----------------------+----------------------------------------------------------------------------------+
-| majority             |                                                                                  |
+| cumulative           | `Cumulative voting`_, where each voter can distribute their vote to up to        |
+|                      | *N* candidates.                                                                  |
 +----------------------+----------------------------------------------------------------------------------+
-| n-of-m               |                                                                                  |
+| majority             | A 1-of-m method where the winner needs more than 50% of the vote to be elected.  |
 +----------------------+----------------------------------------------------------------------------------+
-| plurality            |                                                                                  |
+| n-of-m               | A method where each voter can select up to *N* candidates.                       |
 +----------------------+----------------------------------------------------------------------------------+
-| proportional         |                                                                                  |
+| plurality            | A 1-of-m method where the candidate with the most votes is elected, regardless   |
```

Comment:
Yes, you could say that the winners in approval voting and cumulative voting are also chosen by "plurality."  But my reply here is the same as [my reply](https://github.com/votinginfoproject/vip-specification/pull/184/files#r31302240) in the "majority" case:

Rather than leave things ambiguous (i.e. potentially multiple enum values being applicable to a given contest), I would rather the documentation give clear guidance as to which enum value should be chosen.  For example, for something like approval voting or cumulative voting, I'm betting we can all agree that we would want them to choose "approval" and "cumulative" respectively, rather than "plurality".

Also, just to be clear, by documenting what the enum name "plurality" stands for, that doesn't mean we need to be defining the English _word_ "plurality" in the abstract, general sense.  The enum name is just an abbreviation for the purposes of the VIP spec, and we are free to say what it stands for in the context of the spec (i.e. it doesn't need to mean plurality _literally_).